src/sidebar/util/version-data.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * Build version data object for current session and client
+ *
+ * @param {Object} auth - Object representing auth state and user information
+ * @param {Object} mainFrame - The main frame of the app
+ * @return {Object}
+ */
+const getVersionData = (auth, mainFrame) => {
+  const hasFingerprint =
```

Comment:
Optional chaining and nullish coalescing look like they are going to become Stage 4 (ie. part of the spec) [very soon](https://github.com/tc39/agendas/blob/master/2019/12.md) and already implemented natively in [two browsers](https://www.chromestatus.com/feature/5748330720133120?). On that basis I'd be OK with enabling the relevant Babel plugins so we can use them now. It might come in useful when converting CoffeeScript code as the equivalent feature is used there.